### PR TITLE
remove bash from dependencies

### PR DESCRIPTION
**Pull request by grimreaper**
Remove portable "local" keyword". 
after this, it appears there no bash specific code here. Allow it be used with installing additional software.

### CODE REVIEW

File: src/pb.sh [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+#!/bin/sh
 
 endpoint="${PB_ENDPOINT:-https://ptpb.pw}"
 jq_args="${PB_JSON:--r .url}"
 private="${PB_PRIVATE:-0}"
 
 
 pb_ () {
-  local command filename
+  command filename
```
Comment (buhman) [inline src/pb.sh:9]: Removing `local` here changes the meaning of the line; this executes the `command` shell built-in.

**Comment by buhman** [comment]
I appreciate the contribution!

I see no value in least-common-denominator "shell portability", while changing variable scope from function-local to shell-global is a reduction in correctness.

**Comment by grimreaper** [comment]
That's what I get for doing this from the web without proper testing. The main reason I wanted to do this, is that I have multiple systems that don't have bash, but do have other shells. 

(I'd really love for POSIX to standardize some of the more common stuff, but they are a bit slow-moving)

**Comment by buhman** [comment]
> POSIX to standardize some of the more common stuff

I'd love to see POSIX become completely irrelevant (uncompromising backwards-compatibility in the name of inhibited innovation is a cancer that plagues many popular projects, and POSIX' stagnation doesn't help), but that's just my political opinion.

**Pull request closed without merging**